### ISSUE TITLE
state:Changed: headers mismatch documentation

### ISSUE THREAD

**Issue by sadko4u** [lv2/lv2#45]
In documentation we see:
![image](https://user-images.githubusercontent.com/12834464/84683957-a1783980-af40-11ea-97ca-ea62b3f825e5.png)

While in history:
![image](https://user-images.githubusercontent.com/12834464/84684009-b523a000-af40-11ea-809a-af432fc61bec.png)

And in headers:
```C
#define LV2_STATE__StateChanged      LV2_STATE_PREFIX "StateChanged"       ///< http://lv2plug.in/ns/ext/state#StateChanged

```

**Comment by sadko4u** [comment]
Need to correct **Changed** to **StateChanged**.

**Comment by drobilla** [comment]
Shit.  I really should have caught that.

Yep, it should be StateChanged (the names themselves, ignoring the prefix, should be unique and self-descriptive, but we've screwed this up several times in the past).  Though technically that part (the ttl and the documentation) _is_ the standard, so... this is not great, and fixing it will technically be a spec break.  I think I'll just consider it a mistake (like a typo) and do it anyway, that's what Ardour implements.  I'm not sure about other hosts though...

**Comment by drobilla** [comment]
Fixed in 8e651d2, thanks.
